src/components/Events/useEvents.hooks.ts

```diff
@@ -15,7 +14,7 @@ const getStatus = (startDate: string, endDate: string) => {
 const useEvents = (filterText: string) => {
   const eventsData = useQuery("getEvents");
 
-  const events = eventsData?.map((event) => ({
+  const events = eventsData?.map(event => ({
```

Comment:
Not sure when/why my Prettier settings changed, but maybe we should check in a config file to avoid noise like this.